README.md

```diff
@@ -41,33 +41,33 @@ Good news, you don't have to!  Enter log4cats!  Read on!
 import org.typelevel.log4cats.Logger
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 import cats.effect.Sync
-import cats.implicits._
+import cats.implicits.*
```

Comment:
Should these all be changed to `cats.syntax.all.*`? It's mentioned in these release notes that the latter is preferred
https://github.com/typelevel/cats/releases/tag/v2.2.0

Reply:
I'm indifferent to whether it happens in this PR, but :+1: in general.

Reply:
It's only in 2 places, so it's worth doing now rather than dealing with the overhead of another PR